share/gpodder/extensions/subscription_stats.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+# Show publishing statistics for subscriptions.
+# Released under the same license terms as gPodder itself.
+
+import time
+from time import localtime, strftime
+
+import gpodder
+from gpodder import config
+
+import gi  # isort:skip
+from gi.repository import Gtk, Pango  # isort:skip
+gi.require_version('Gtk', '3.0')  # isort:skip
+
+_ = gpodder.gettext
+
+__title__ = _('Subscription Statistics')
+__description__ = _('Show publishing statistics for subscriptions.')
+__only_for__ = 'gtk'
+__doc__ = 'https://gpodder.github.io/docs/extensions/subscription_stats.html'
+__authors__ = 'Brand Huntsman <http://qzx.com/mail/> and \
+               Nuno Dias <Nuno.Dias+gpodder@gmail.com>'
+
+
+class gPodderExtension:
+    def __init__(self, container):
+        self.container = container
+
+    def on_ui_object_available(self, name, ui_object):
+        if name == 'gpodder-gtk':
+            self.gpodder = ui_object
+
+    def on_create_menu(self):
+        # extras menu
+        return [(_("Subscription Statistics"), self.open_dialog)]
+
+    def add_page(self, notebook, category, channels):
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        store = Gtk.ListStore(str, float, str, str, int)
+        for average, name, edate, paused in channels:
+            last = strftime('%x', localtime(edate))
+            store.append([
+                ('%.1f' % round(average, 1)) if average > 0 else '?',
+                average, ('‖ ' if paused else '') + name, last, edate,
+            ])
+
+        tree = Gtk.TreeView(model=store)
+        scrolled.add(tree)
+
+        dayscell = Gtk.CellRendererText()
+        dayscell.set_property('xalign', 0.5)
+        dayscolumn = Gtk.TreeViewColumn(_('Days'))
+        dayscolumn.set_sort_column_id(1)
+        dayscolumn.pack_start(dayscell, True)
+        dayscolumn.add_attribute(dayscell, 'text', 0)
+        tree.append_column(dayscolumn)
+
+        channelcell = Gtk.CellRendererText()
+        channelcell.set_property('ellipsize', Pango.EllipsizeMode.END)
+        channelcolumn = Gtk.TreeViewColumn(_('Channel'))
+        channelcolumn.set_sort_column_id(2)
+        channelcolumn.pack_start(channelcell, True)
+        channelcolumn.add_attribute(channelcell, 'text', 2)
+        channelcolumn.set_expand(True)
+        tree.append_column(channelcolumn)
+
+        lastcell = Gtk.CellRendererText()
+        lastcolumn = Gtk.TreeViewColumn(_('Last Updated'))
```

Comment:
The header in this column is now usually wider than the values. Using just 'Updated' as column header would take less space.

Reply:
Ok, chnaged.